airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -804,6 +804,13 @@ def generate_scd_type_2_model(self, from_table: str, column_names: Dict[str, Tup
             cdc_cols += f", {cast_begin}{col_cdc_log_pos}{cast_as}" + "{{ dbt_utils.type_string() }}" + f"{cast_end}"
             quoted_cdc_cols += f", {quoted_col_cdc_log_pos}"
 
+        if "_ab_cdc_lsn" in column_names.keys():
+            col_cdc_lsn = self.name_transformer.normalize_column_name("_ab_cdc_lsn")
+            quoted_col_cdc_lsn = self.name_transformer.normalize_column_name("_ab_cdc_lsn", in_jinja=True)
+            cdc_updated_order_pattern += f"\n            {col_cdc_lsn} desc,"
+            cdc_cols += f", {cast_begin}{col_cdc_lsn}{cast_as}" + "{{ dbt_utils.type_string() }}" + f"{cast_end}"
```

Comment:
this is fine, the mixture of f strings and non f strings which represent jinja replacement tokens threw me off. Random non required suggestions
```suggestion
            cdc_cols += "".join([", ", cast_begin, col_cdc_lsn, cast_as, "{{ dbt_utils.type_string() }}", cast_end])
```